### PR TITLE
example(LLM): add musicgen example

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
- cloud resources:
  - dataset: https://cloud.starwhale.cn/projects/23/datasets/61/versions/89/files
  - model: https://cloud.starwhale.cn/projects/23/models/28
  - runtime:  https://cloud.starwhale.cn/projects/23/runtimes/23/versions/60/overview
- cloud evaluation:
  - https://cloud.starwhale.cn/projects/23/evaluations/206/results  
- muscigen mini dataset:
  image
- model run for standalone
  image

## Modules
- [x] Others

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: example/LLM/musicgen/build.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        prepare_build_model_package,
+    )
+    from evaluation import music_predict
+
+debug.init_logger(4)
+
+
+def build_starwhale_model(model_name: str) -> None:
+    """
+    Build starwhale model from raw llama model and delta vicuna model.
```
Comment (anda-ren) [inline example/LLM/musicgen/build.py:30]: the comment seems wrong

File: example/LLM/musicgen/dataset.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "An 80s driving pop song with heavy drums and synth pads in the background",
+    "	An energetic hip-hop music piece, with synth sounds and strong bass. There is a rhythmic hi-hat patten in the drums.",
+]
+
+
+def build_dataset() -> None:
+    print("Building musicgen dataset...")
+    with dataset("musicgen-mini") as ds:
+        for desc in desc_samples:
+            ds.append({"desc": desc})
```
[the remark this answers is not in the document]
Comment (tianweidut) [reply example/LLM/musicgen/dataset.py:40]: The currtent evaluation code only supports `text -> audio`.  `text +melody(audio) -> audio` feature will be added in another pr, so the dataset only includes text column.

File: example/LLM/musicgen/build.py (hunk 1 above)
Comment (tianweidut) [reply example/LLM/musicgen/build.py:30]: fixing

**Review by anda-ren — APPROVED** [review]
LGTM

**Pull request merged**